package.json

```diff
@@ -138,6 +140,7 @@
         "webpack-cli": "^5.0.2"
     },
     "dependencies": {
-        "@babel/runtime": "^7.6.0"
+        "@babel/runtime": "^7.23.2",
+        "core-js": "^3.33.0"
```

Comment:
No it doesn't.  All of the sites I read say to put it in dependencies because it is used by @babel/runtime.  However, since we aren't using it in production but only for tests, I suppose it is okay to put in dev-dependencies.  Moved it there and the test and prod builds look identical